src/utils/localisation/exportLanguages.ts

```diff
@@ -53,6 +53,8 @@ export const exportLanguages = async (
     const row3code = ['Code:', '', ...languageOptions.map((opt) => opt.code)]
     const row4Flag = ['Flag:', '', ...languageOptions.map((opt) => opt.flag)]
     const row5Enabled = ['Enabled?', '', ...languageOptions.map((opt) => opt.enabled)]
+    const row6Locale = ['Locale:', '', ...languageOptions.map((opt) => opt.locale)]
```

Comment:
I think the "Enabled" row should always be last.

Reply:
Hmmmmm ok